storage/reads/gen.go

```diff
@@ -1 +1,3 @@
 package reads
+
+//go:generate env GO111MODULE=on go run github.com/benbjohnson/tmpl -data=@array_cursor.gen.go.tmpldata array_cursor.gen.go.tmpl
```

Comment:
Remove this. I removed generate statements for code I copied over to avoid interfering with the build or modifying the files.